Azure.ResourceManager.Core/ArmResourceOperations.cs

```diff
@@ -11,13 +11,13 @@ namespace Azure.ResourceManager.Core
     public class ArmResourceOperations : ResourceOperationsBase<ArmResourceOperations, ArmResource>,
         ITaggable<ArmResourceOperations, ArmResource>, IDeletableResource<ArmResourceOperations, ArmResource>
     {
-        public ArmResourceOperations(AzureResourceManagerClientContext context, ResourceIdentifier id, AzureResourceManagerClientOptions clientOption)
-            : base(context, id, clientOption)
+        public ArmResourceOperations(AzureResourceManagerClientContext context, ResourceIdentifier id)
```

Comment:
Should have specified in the ADO item but we should follow the standard naming convention of [ClientName]Options instead of [ClientName]Context